doc/Doxyfile

```diff
@@ -15,32 +15,30 @@
 DOXYFILE_ENCODING      = UTF-8
 PROJECT_NAME           = "TDMS"
 PROJECT_BRIEF          = "Time Domain Maxwell Solver"
-# Blank == no logo
-PROJECT_LOGO           =
-# Blank == Write output to current dir.
-OUTPUT_DIRECTORY       =
-# Input directories to search for files
-INPUT                  = tdms/include/ tdms/src/ tdms/tests/unit/ tdms/tests/include tdms/tests/system
-# Recurse through input directories to find subdirectories
-RECURSIVE              = YES
-FILE_PATTERNS          = *.cpp \
-                         *.h \
-                         *.py
+PROJECT_LOGO           = # Blank == No logo.
+OUTPUT_DIRECTORY       = # Blank == Write output to current dir.
+INPUT                  = tdms/include tdms/src tdms/tests/unit
+FILE_PATTERNS          = *.cpp *.h *.py
```

Comment:
It might seem a bit odd to people that we say TDMS is a hybrid `MATLAB` and `C++` software, yet we don't include the `.m` files in Doxygen?

I'm not saying we _should_ do this (as `.m` is also a legacy `C` format or something right?) but if we're encouraging people to use the `MATLAB` scripts in the examples we probably should explain how they operate somewhere.

Reply:
I do agree. Googling around it seems people have managed to get MATLAB docs with both Doxygen and Sphinx. 

The former being a bit weird... I think we want a 'filter' to convert the functions into something like C (which Doxygen understands) then run Doxygen. Worth an issue though (even though it's more like "nice to have" for our End of ARC milestone).

* #266